test/bin/run

```diff
@@ -4,7 +4,13 @@ cd "$(dirname "$0")/../"
 
 set -e
 
-. ../nginx-version-info
+if [ $(echo ${RESTY_TEST}) = "ON" ] ; then
+    . ../openresty-version-info
+    cp ../test/cases/auto_propagation/openresty_base_conf/* ../test/cases/auto_propagation/conf/
+else
+    . ../nginx-version-info
+    cp ../test/cases/auto_propagation/nginx_base_conf/* ../test/cases/auto_propagation/conf/
+fi
```

Comment:
[suggestion] I would suggest a solution that do not mess with the user filesystem.